backend/node/genvm/base.py

```diff
@@ -13,6 +13,16 @@
 from backend.node.genvm.code_enforcement import code_enforcement_check
 
 
+class ContractRunner:
+    def __init__(self, mode: str, node_config: dict):
+        self.mode = mode
+        self.node_config = node_config
+        self.from_address = None
```

Comment:
when is `None` a valid value for `from_address`?

Reply:
sorry, this was unanswered: when you need to get a contract schema or read from a contract state, you are not sending a transaction thus the from_address is None.
More info: https://stackoverflow.com/a/51850115/637623